### DISCUSSION TITLE
Use force for fill create and update removing the need for $fillable

### DISCUSSION THREAD

**Discussion by steks89** [filamentphp/filament#3731] [Ideas]
Suggestion:

As we are hardcoding our forms, it is obvious that we want our fields to be fillable.

Filament should be using force for fill, create and update removing the need for $fillable on ours models as I might not want it to be fillable on other parts of my code.

`return $this->getModel()::forceCreate($data);`

` $record->forceFill($data);
 $record->update();`

**Comment by danharrin** [comment]
Fair enough, all data is validated anyway. PR?

**Reply by steks89** [reply]
Hey, thank your for your reply.
I am happy to make a PR.
